resources/views/poller-group/index.blade.php

```diff
@@ -0,0 +1,19 @@
+@extends('layouts.librenmsv1')
+
+@section('content')
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-12">
+      <ul class="nav nav-tabs">
+        @foreach($navbar as $tab)
```

Comment:
I would hardcode the tabs and just pass the active tab in to blade.

Reply:
done